src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -301,8 +301,15 @@ private Object parseValue(StaplerRequest request) throws ServletException, IOExc
             }
         }
 
-        // TODO: perhaps we should return a different object to allow the workflow to look up
-        // who approved it, etc?
+        // If requested, add the approver to the list of returned parameters.
+        String submitter = input.getSubmitter();
```

Comment:
No, this is just the definition of the `submitter` parameter in the script—includes no runtime information. Your test does not catch the mistake because it is hard-coding a single username, rather than using a comma-separated list or a `SecurityRealm` group name.

You meant to check `Jenkins.getAuthentication`; see elsewhere in this class.